### PR TITLE
Webpage

### PR DESCRIPTION
**Pull request by ricieritab**

### CODE REVIEW

File: scripts/main.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
+// main.js
+
+const allSquares = elementsByClassName("square");
+
+function main() {
+    allSquares
+        .forEach(addGameClickListeners);
+
```
Comment (raynelle03) [inline scripts/main.js:8]: space here?

File: scripts/main.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+main();
+
+// Private Properties
+
+let _gameOver = false;
+let _lastPlayer;
+let _playerOneGame = [];
+let _playerTwoGame = [];
+
```
Comment (raynelle03) [inline scripts/main.js:26]: Just out of curiosity, what's the purpose of having the _ infront of the variable and functions? Is that a syntax style thing?

File: scripts/main.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+    }
+
+    _updateLastPlayer(player);
+    _savePositionToPlayerGame(parseInt(position));
+
+    if (_isMatchOver()) {
+        _evaluateMatch();
+        return;
+    }
+
```
Comment (raynelle03) [inline scripts/main.js:138]: Space here

**Review by raynelle03 — APPROVED** [review]

File: scripts/main.js (hunk 1 above)
Comment (ricieritab) [reply scripts/main.js:8]: I personally prefer currying methods that works in a chain in a new indentation level

File: scripts/main.js (hunk 2 above)
Comment (ricieritab) [reply scripts/main.js:26]: An _ prefixing the object or the function indicates it is private

**Pull request merged**